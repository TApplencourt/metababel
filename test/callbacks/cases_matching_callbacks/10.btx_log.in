event_1: { pf_1 = 0 }
event_2: { pf_1 = 0 }
event_3: { pf_1 = "0" }
