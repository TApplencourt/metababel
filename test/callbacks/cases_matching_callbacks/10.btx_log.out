event_4: { pf_1 = 0 }
event_4: { pf_1 = 0 }
event_4: { pf_1 = 0 }
